### PR TITLE
New avoidance of food intake pattern, tsv, and add related terms

### PR DESCRIPTION
**Pull request by LCCarmody**
@matentzn Tried to write a pattern and tsv file.  I don't recall how to change the imports so it brings in Foodon

### CODE REVIEW

File: src/patterns/data/manual/avoidance_of_food_intake.tsv.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+defined_class	medical_action	medical_action_label	food_material	food_material_label	created_by	date	defined_class_definition	rdfs:comment
```
Comment (matentzn) [inline src/patterns/data/manual/avoidance_of_food_intake.tsv.txt]: filename does not need .txt in the end

File: src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+pattern_name: avoidance_of_food
```
Comment (matentzn) [inline src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml]: pattern name should match filename and iri!

File: src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+pattern_iri: http://purl.obolibrary.org/obo/maxo/patterns/avoidance_of_medical_action.yaml
```
Comment (matentzn) [inline src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml]: wrong iri, should match filename

File: src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+description: ""
```
Comment (matentzn) [inline src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml]: There should be a reasonable explanation here along the line of what was discussed in the ticket.

File: src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    vars:
+     - created_by
+def:
+  text: "A medical requirement of avoiding the %s of %s."
+  vars:
+    - medical_action
+    - food_material
+
+equivalentTo:
+  text: "'avoidance' and ('avoided_medical_action' some ('%s' and has_input some %s))"
```
Comment (matentzn) [inline src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml]: A medical action is anything that alleviates medical problems. How would burger eating (so non-medical actions like consuming unhealthy stuff) fit in the conceptual model here? Maybe we should rethink "avoiding" patterns actually refer to actions in general, not just medical ones.. Avoid smoking etc. Smoking is clearly not a medical action. I think you can go ahead and keep working on this, but my leave the pull request open an maybe put it on the agenda for our next meeting? This should not keep you from working, I think you can go ahead and collect the relevant foodon terms in the TSV.

File: src/patterns/data/manual/avoidance_of_food_intake.tsv.txt (hunk 1 above)
Comment (LCCarmody) [reply src/patterns/data/manual/avoidance_of_food_intake.tsv.txt]: removed

**Comment by LCCarmody** [comment]
Updated all edits. Let me know if this is okay.

**Pull request merged**